### ISSUE TITLE
Group environment variable by application scope

### ISSUE THREAD

**Issue by SeanFeldman** [HTBox/allReady#562]
When working with environment variables, need to take into account that different apps can use similar environment variable and there's a chance of collision. Example:
* `Data:Storage:AzureStorage`
* `Authentication:SendGrid:UserName`
* `ApplicationInsights:InstrumentationKey`
* etc

Suggestion is to prefix with application scope. Example:
* `AllReady:Data:Storage:AzureStorage`
* `AllReady:Authentication:SendGrid:UserName`
* `AllReady:ApplicationInsights:InstrumentationKey`
* etc

**Comment by tonysurma** [comment]
@dpaquette @MisterJames thoughts? :thought_balloon:

**Comment by stevejgordon** [comment]
@tonysurma I'd agree this is probably worthwhile and I suspect quite a simple change. Happy to work on a PR if @MisterJames also thinks there's value in doing this?

**Comment by tonysurma** [comment]
@MisterJames @dpaquette thoughts?

**Comment by dpaquette** [comment]
Yeah we should do this but I would consider it fairly low priority. It should be relatively easy to do but we have to consider that it will require changes to any deployed environments and any local user secrets that existing developers have setup.
